website/pages/docs/nia/installation/install.mdx

```diff
@@ -6,13 +6,20 @@ description: >-
   Consul-Terraform-Sync is a daemon that runs alongside Consul. Consul-Terraform-Sync is not included with the Consul binary and will need to be installed separately.
 ---
 
-# Installing Consul
+# Install Consul and Consul-Terraform-Sync
 
-Consul-Terraform-Sync is a daemon that runs alongside Consul, similar to other Consul ecosystem tools like Consul Template. Consul-Terraform-Sync is not included with the Consul binary and will need to be installed separately. First, we recommend installing Consul. For simplicity, we are using a local Consul agent in development mode which can be started with `consul agent -dev`. To quickly set up a local Consul agent, refer to the getting started [guide](https://learn.hashicorp.com/tutorials/consul/get-started-install). The Consul agent must be running to dynamically update network devices.
+## Installing Consul
 
-# Installing Consul-Terraform-Sync
+Consul-Terraform-Sync is a daemon that runs alongside Consul, similar to other Consul ecosystem tools like Consul Template. Consul-Terraform-Sync is not included with the Consul binary and needs to be installed separately.
 
-To install Consul-Terraform-Sync, find the [appropriate package](https://releases.hashicorp.com/) for your system and download it as a zip archive. Unzip the package to extract the binary named consul-terraform-sync. Move the consul-terraform-sync binary to a location available on your PATH.
+To install a local Consul agent, refer to the [Getting Started: Install Consul Tutorial](https://learn.hashicorp.com/tutorials/consul/get-started-install).
+
+The Consul agent must be running in order to dynamically update network devices. To run the local Consul agent, for simplicity, run in development mode which can be started with `consul agent -dev`. For more details on running Consul, see [Run a Consul Cluster](/docs/nia/installation/requirements#run-a-consul-cluster).
```

Comment:
This reads a bit too strong that we expect running Consul in dev mode, when it can be an option or connect to an already existing Consul cluster. Can you revert closer to the original verbiage?

Reply:
Thinking about this at a high level, I think installing Consul is a weird first step for the Install page for Sync. Consul agent is a requirement. Can you move this portion to fit the req page instead?

Reply:
I see what you're saying. This is the installing sync page vs. installing everything page. I'll move it. Just to continue the conversation about the content though, what are your thoughts on:

> To run the local Consul agent, you can run Consul in development mode which can be started with `consul agent -dev` for simplicity.

Re: "Connect to your Consul cluster". I think that makes sense. I'll add it.

Reply:
Yeah that reads well to me!

Sweet, thanks for looking into shifting things around -- I know you were following the original organization.